docs/src/main/sphinx/installation/cli.rst

```diff
@@ -13,28 +13,182 @@ Requirements
+  (8 rows)
 
-The CLI uses the HTTP protocol and the
-:doc:`Trino client REST API </develop/client-protocol>` to communicate
-with Trino.
+You can also set the default catalog and schema with the :doc:`/sql/use`
+statement.
+
+.. code-block:: text
+
+  trino> USE tpch.tiny;
+  USE
+  trino:tiny>
+
+Many other options are available to further configure how to start the CLI in
+interactive mode:
+
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``--catalog``
+    - Sets the default catalog. You can change the default catalog and schema
+      with :doc:`/sql/use`.
+  * - ``--client-info``
+    - Adds extra information about the client.
+  * - ``--client-request-timeout``
+    - Sets the duration for query processing, after which, the client request is
+      terminated. Defaults to ``2m``.
+  * - ``--client-tags``
+    - Adds extra tags information about the client and the CLI user. The tags
+      can be used as input for :doc:`/admin/resource-groups`.
+  * - ``--debug``
+    - Enables display of debug information during CLI usage for
+      :ref:`cli-troubleshooting`. Displays more information about query
+      processing statistics.
+  * - ``--disable-compression``
+    - Disables compression of query results.
+  * - ``--editing-mode``
+    - Sets key bindings in the CLI to be compatible with VI or
+      EMACS editors. Defaults to ``EMACS``.
+  * - ``--http-proxy``
+    - Configures the URL of the HTTP proxy to connect to Trino.
+  * - ``--network-logging``
+    - Configures the level of detail provided for network logging of the CLI.
+      Defaults to ``NONE``, other options are ``BASIC``, ``HEADERS``, or
+      ``BODY``.
+  * - ``--password``
+    - Prompts for a password. You can set the ``TRINO_PASSWORD`` environment
+      variable with the password value to avoid the prompt. For more
+      information, see :ref:`cli-username-password-auth`.
+  * - ``--schema``
+    - Sets the default schema. You can change the default catalog and schema
+      with :doc:`/sql/use`.
+  * - ``--server``
+    - The HTTP/HTTPS address and port of the Trino coordinator. The port must be
+      set to the port the Trino coordinator is listening for connections on.
+      Trino server location defaults to ``http://localhost:8080``.
+  * - ``--session``
+    - Sets one or multiple :ref:`session properties
+      <session-properties-definition>`. Property can be used multiple times with
+      the format ``key=value``.
+  * - ``--socks-proxy``
+    - Configures the URL of the SOCKS proxy to connect to Trino.
+  * - ``--source``
+    - Specifies the name of the application or source connecting to Trino.
+      Defaults to ``trino-cli``. The value can be used as input for
+      :doc:`/admin/resource-groups`.
+  * - ``--timezone``
+    - Sets session time zone. Defaults to ``America/New_York``.
+  * - ``--trace-token``
+    - Sets a token value for correlating requests and responses during
+      debugging.
```

Comment:
Let's remove this since it's not something anyone uses and we don't have any supporting documentation guiding anyone on how to actually make use of this.

Reply:
ok